openquake/qa_tests_data/scenario_risk/case_1/expected/losses_by_event.csv

```diff
@@ -7,6 +7,5 @@ event_id,structural,rlz_id
 4,3.42458E+02,0
 5,2.52500E+02,0
 6,4.81053E+02,0
```

Comment:
Should we print the event with zero losses in the exported file? A scenario usually has a limited number of events, so it won't be an issue printing the losses for all events. Suppressing the zero loss events is more useful in the event-based outputs which might have oversized tables.

Reply:
The point is that Helen (see the original issue) wants to be able to compare scenario_risk with ebrisk, so we must use the same approach, i.e. discard zero losses events in both calculators. Notice that the "events" table always contains all events and does not discard them at all. It is only the event loss table that can miss events.

Reply:
Also, if most scenario_risk calculation are small, there is no need to set `minimum_asset_loss` and no risk of losing events ;-)